lib/matplotlib/cbook/__init__.py

```diff
@@ -2242,7 +2242,7 @@ def type_name(tp):
                     type_name(type(v))))
 
 
-def _check_in_list(_values, **kwargs):
+def _check_in_list(_values, *, _print_supported_values=True, **kwargs):
```

Comment:
The parameter doesn't need to be private; the whole function is already,

Reply:
Ah, never mind, I guess this is to avoid conflict with anything passed with `kwargs`.

Please update the docstring though.

Reply:
I just updated the docstring. Let me know if I made any mistake with the wording/conventions. Thanks!

Reply:
It looks good to me.